tabpy-server/tabpy_server/common/config.py.template

```diff
@@ -2,4 +2,5 @@ TABPY_QUERY_OBJECT_PATH = '/tmp/query_objects'
 TABPY_PORT = 9004
 TABPY_SERVER_VERSION = 'Alpha'
 TABPY_STATE_PATH = './'
-TABPY_BIND_IP = '0.0.0.0'
\ No newline at end of file
+TABPY_BIND_IP = '0.0.0.0'
+TABPY_LOG_LEVEL = 'INFO'
```

Comment:
Is the intent to change the level to INFO or should it be left at ERROR as the default?